e2e-tests/cypress/integration/appellant-submission-decision-letter/appellant-submission-decision-letter.js

```diff
@@ -25,7 +25,12 @@ Then('user can see that no decision letter file is submitted', (reason) => {
   cy.confirmDecisionLetterRejectedBecause('Upload the decision letter');
 });
 
-Then('user can see that the decision letter file {string} "is" submitted', (filename) => {
+Then('user can see that the decision letter file {string} is submitted', (filename) => {
+  cy.saveAndContinue();
```

Comment:
I think the cy.saveAndContinue function might better be in the When step because it is the trigger event/action.